### PR TITLE
add GET Protocol fee adapter

### PR DESCRIPTION
**Pull request by efe-get-protocol**
Name (to be shown on DefiLlama):
GET Protocol

Twitter Link:
https://twitter.com/GetProtocol

List of audit links if any:
https://www.get-protocol.io/files/GET-Protocol-Token-Sale-Matthew-Di-Ferrante.pdf
https://reports.yacademy.dev/docs/12-2022-GET-Protocol-Staking/

Website Link:
https://www.get-protocol.io/

Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://files.readme.io/8518f89-GET_Protocol_-_Main_Logo_No_Text.svg

Current TVL:
n/a

Chain:
Ethereum and Polygon

Coingecko ID (so your TVL can appear on Coingecko):
get-token

Coinmarketcap ID (so your TVL can appear on Coinmarketcap):
get-protocol

Short Description (to be shown on DefiLlama):
GET Protocol provides NFT ticketing infrastructure to put real NFT ticket distribution in the hands of event organisers and ticketing companies.

Token address and ticker if any:
0x8a854288a5976036A725879164Ca3e91d30c6A1B Ethereum
0xdb725f82818De83e99F1dAc22A9b5B51d3d04DD4 Polygon 
Ticker: GET

Category (full list at https://defillama.com/categories) *Please choose only one:
RWA

Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
n/a

forked from (Does your project originate from another project):
n/a

methodology (for fees):
Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol

### CODE REVIEW

**Comment by llamatester** [comment]
The get-protocol.ts adapter exports: 
        
 
 _______________________________________
    Fees for 26/3/2023
    _______________________________________
    
    Wrong value NaN with key dailyProtocolRevenue
    POLYGON 👇
    Backfill start time: 1/9/2021
    Methodology: Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol
    Timestamp: 1679875198
    Total fees: 506739
    Daily fees: 220
    
    
    
    
    Done in 4.58s.

**Comment by llamatester** [comment]
The get-protocol.ts adapter exports: 
        
 
 _______________________________________
    Fees for 26/3/2023
    _______________________________________
    
    42
    POLYGON 👇
    Backfill start time: 1/9/2021
    Methodology: Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol
    Timestamp: 1679875198
    Total fees: 506764.03774029826
    Daily fees: 245.4539346378567
    Daily protocol revenue: 42
    
    
    
    
    Done in 3.91s.

**Comment by llamatester** [comment]
The get-protocol.ts adapter exports: 
        
 
 _______________________________________
    Fees for 26/3/2023
    _______________________________________
    
    42
    POLYGON 👇
    Backfill start time: 1/9/2021
    Methodology: Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol
    Timestamp: 1679875198
    Total fees: 506764.03774029826
    Daily fees: 245.4539346378567
    Daily protocol revenue: 42
    
    
    
    
    Done in 4.10s.

**Comment by llamatester** [comment]
The get-protocol.ts adapter exports: 
        
 
 _______________________________________
    Fees for 26/3/2023
    _______________________________________
    
    POLYGON 👇
    Backfill start time: 1/9/2021
    Methodology: Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol
    Timestamp: 1679875198
    Total fees: 506764.03774029826
    Daily fees: 245.4539346378567
    Daily protocol revenue: 42
    
    
    
    
    Done in 3.81s.

File: fees/get-protocol.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      parseFloat(graphRes.protocolDays[0].reservedFuel) * getPrice;
+
+    const finalFeeAllTime =
+      parseFloat(graphResAllTime.protocol.reservedFuel) * getPrice;
+
+    //GUTS fees
+    const gutsFeesDaily =
+      parseFloat(graphGutsFees.integratorDays[0].reservedFuel) * getPrice;
+
+    const dailyRevenue = Math.trunc((finalDailyFee - gutsFeesDaily) * 0.8);
```
Comment (dtmkeng) [inline fees/get-protocol.ts]: could you remove `Math.trunc` ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The get-protocol.ts adapter exports: 
        
 
 _______________________________________
    Fees for 26/3/2023
    _______________________________________
    
    POLYGON 👇
    Backfill start time: 1/9/2021
    Methodology: Ticketeers pay an on-chain fee in GET for every ticket that they sell through GET Protocol
    Timestamp: 1679875198
    Total fees: 505866
    Daily fees: 347
    Daily protocol revenue: 55
    
    
    
    
    Done in 5.87s.

File: fees/get-protocol.ts (hunk 1 above)
Comment (efe-onchain) [reply fees/get-protocol.ts]: done.

**Pull request merged**